### ISSUE TITLE
back button is in wrong position

### ISSUE THREAD

**Issue by zoff99** [HeinrichReimer/material-intro#123]
on the first slide the back button is to low. when i slide to the next slide its ok again.
very strange.
i am using Release 1.5.7

![screenshot_20160901-095237](https://cloud.githubusercontent.com/assets/6833516/18202821/e26bfc44-7112-11e6-987a-29b120bdcf22.png)

**Comment by daniel-stoneuk** [comment]
Was this a one off, or does it happen every time?

**Comment by zoff99** [comment]
every time on the first slide.
now i changed a bit in your code and its work. i think its related to View.GONE and View.INVISIBLE
but not sure

**Comment by daniel-stoneuk** [comment]
Right, then hopefully this is fixed as I made that change in a recent
commit. If you want to use the latest version, replace the dependency with
this:
com.heinrichreimersoftware:material-intro:c68005f
That will use the latest commit, but will not change if something else or a
new bug is added to the library.

**Comment by HeinrichReimer** [comment]
@zoff99, @daniel-stoneuk I will release version 1.5.9 this weekend so you could don't have to use a snapshot release then.
